tests/integration/test_rename_column/test.py

```diff
@@ -225,35 +225,53 @@ def select(
 def rename_column(
     node, table_name, name, new_name, iterations=1, ignore_exception=False
 ):
-    for i in range(iterations):
+    i = 0
+    while True:
+        i += 1
         try:
             node.query(
                 "ALTER TABLE {table_name} RENAME COLUMN {name} to {new_name}".format(
                     table_name=table_name, name=name, new_name=new_name
                 )
             )
         except QueryRuntimeException as ex:
+            if "Coordination::Exception" in str(ex):
```

Comment:
shouldn't we be more specific here - maybe check for "Coordination::Exception: Session expired"?

Reply:
I think no, because there can be several similar errors like "Connection loss" and "Operation timeout", I just don't want to make complex grep.